Cloud Application Manager/Getting Started/register-existing-aws-windows-instance.md

```diff
@@ -56,7 +56,7 @@ Provide any details for the instance in the fields provided.
 
 Click **Register** to register the instance.
 
-If you didn't provide the *Private Key*, *Windows Username* and *Password* fields *Cloud Application Manager* returns an error message because the agent installation script that allows *CenturyLink* to register the instance is not yet executed on this instance.
+If you didn't provide the *Private Key*, *Windows Username* and *Password* fields, or the instance IP address is not accesible from the internet, *Cloud Application Manager* returns an error message because the agent installation script that allows *Cloud Application Manager* to register the instance is not yet executed on this instance. You can then proceed to the manual instalation process described below.
```

Comment:
Could it be possible to replace second Cloud Application Manager with CAM for this not to be repeated a few words later?

Same for instance in.."to register the instance is not yet executed on this instance" --> "to register the instance is not yet executed on it"

Reply:
I reworded it. We should not use acronyms in the KB articles